spark/v3.3/spark/src/test/java/com/netease/arctic/spark/writer/TestSparkWriter.java

```diff
@@ -0,0 +1,197 @@
+package com.netease.arctic.spark.writer;
+
+
+import com.netease.arctic.BasicTableTestHelper;
+import com.netease.arctic.TableTestHelper;
+import com.netease.arctic.ams.api.TableFormat;
+import com.netease.arctic.catalog.BasicCatalogTestHelper;
+import com.netease.arctic.catalog.CatalogTestHelper;
+import com.netease.arctic.catalog.TableTestBase;
+import com.netease.arctic.hive.TestHMS;
+import com.netease.arctic.hive.catalog.HiveCatalogTestHelper;
+import com.netease.arctic.hive.catalog.HiveTableTestHelper;
+import com.netease.arctic.spark.reader.SparkParquetReaders;
+import com.netease.arctic.table.ArcticTable;
+import org.apache.iceberg.DataFile;
+import org.apache.iceberg.Files;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.io.CloseableIterable;
+import org.apache.iceberg.orc.ORC;
+import org.apache.iceberg.parquet.AdaptHiveParquet;
+import org.apache.iceberg.relocated.com.google.common.collect.Iterators;
+import org.apache.iceberg.spark.SparkSchemaUtil;
+import org.apache.iceberg.spark.data.SparkOrcReader;
+import org.apache.spark.sql.catalyst.InternalRow;
+import org.apache.spark.sql.catalyst.expressions.GenericInternalRow;
+import org.apache.spark.sql.connector.write.DataWriter;
+import org.apache.spark.sql.connector.write.LogicalWriteInfoImpl;
+import org.apache.spark.sql.connector.write.Write;
+import org.apache.spark.sql.types.Decimal;
+import org.apache.spark.sql.types.StructType;
+import org.apache.spark.sql.util.CaseInsensitiveStringMap;
+import org.apache.spark.unsafe.types.UTF8String;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import static com.netease.arctic.table.TableProperties.DEFAULT_FILE_FORMAT_ORC;
+
+@RunWith(Parameterized.class)
+public class TestSparkWriter extends TableTestBase {
+  @ClassRule
+  public static TestHMS TEST_HMS = new TestHMS();
+
+  public TestSparkWriter(CatalogTestHelper catalogTestHelper,
+      TableTestHelper tableTestHelper) {
```

Comment:
Spark module using junit5 framework for unit test. so this case will not be called.

Reply:
> Spark module using junit5 framework for unit test. so this case will not be called.

Thanks for your review, I had changed it to junit5.

> Spark module using junit5 framework for unit test. so this case will not be called.

Thanks for your review, I had change it to junit5